sadcowpng/sharedspaces/templates/sharedspaces/account.html

```diff
@@ -27,18 +27,24 @@ <h1>Welcome, {{ user.get_username }}!</h1>
             <h3>My Listings</h3>
             <div class="row">
                 {% for spaces in space %}
-                    <div class="card">
-                        <div class="card-body">
-                            <h5 class="card-title">{{ spaces.space_name }}
-                            {% if spaces.space_open %}
-                                <span class="badge badge-success">Open</span>
-                            {% else %}
-                                <span class="badge badge-danger">Closed</span>
-                            {% endif %}
-                            </h5>
-                            <p class="card-text">{{ spaces.space_description }}</p>
-                            <a href="{% url 'date_time' spaces.id %}" class="btn btn-primary">Edit Time</a>
-                            <a href="{% url 'update_space' spaces.id %}" class="btn btn-primary">Update Space</a>
+                    <div class="col-sm-5">
+                        <div class="card">
+                            <div class="card-body">
+                                <h5 class="card-title">{{ spaces.space_name }} 
+                                  {% if spaces.space_open %}
+                                    <span class="badge badge-success">Open</span>
+                                  {% else %}
+                                    <span class="badge badge-danger">Closed</span>
+                                  {% endif %}</h5>
+                                <p class="card-text">{{ spaces.space_description }}</p>
+                                {% for tag in spaces.space_tags.all %}
+                                    <button type="button" class="badge rounded-pill"
+                                            onclick="window.location.href='{% url 'tagged_spaces' tag %}'">{{ tag }}</button>
```

Comment:
Oh, I think because I used the exact class the open-closed should be fine. Nice use of badges.